packages/11ty/_layouts/search-index.11ty.js

```diff
@@ -1,6 +1,6 @@
-module.exports = function ({ pages }) {
+module.exports = function ({ collections }) {
   return JSON.stringify(
-    pages.map((page) => {
+    collections.html.map((page) => {
```

Comment:
It would be fantastic if we could implement the search to also show results in the downloadable PDF though I may be making up fancy features, which no one really needs.